src/system/AdminModule/Controller/AdminInterfaceController.php

```diff
@@ -275,6 +275,8 @@ public function menuAction()
             $moduleNames[$key] = $module['displayname'];
         }
         array_multisort($moduleNames, SORT_ASC, $adminModules);
+        $baseUrl = System::getBaseUrl();
```

Comment:
can you please modify this to either get the information from the Kernel (for full path stuff) or Request (for web path stuff)? It will just save fixing this again later.